common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -14,10 +14,9 @@
 
 define(
 'video/01_initialize.js',
-['video/03_video_player.js', 'video/00_i18n.js'],
+['video/03_video_player.js', 'video/00_i18n.js', 'moment'],
 function (VideoPlayer, i18n) {
     var moment = window.moment;
```

Comment:
@andy-armstrong When we are adding it as third parameter its returning undefined, thats why I used it via window. I read somewhere i don't remember where that moment initialize itself on window object if we don't use ```noGlobal: true``` otherwise it will return undefined.